test/test_art.py

```diff
@@ -209,10 +209,10 @@ def mock_response(self, url, body):
     def test_aao_scraper_finds_image(self):
         body = b"""
         <br />
-        <a href="TARGET_URL" title="View larger image"
-           class="thickbox" style="color: #7E9DA2; text-decoration:none;">
-        <img src="http://www.albumart.org/images/zoom-icon.jpg"
-             alt="View larger image" width="17" height="15"  border="0"/></a>
+        <a href=\"TARGET_URL\" title=\"View larger image\"
+           class=\"thickbox\" style=\"color: #7E9DA2; text-decoration:none;\">
+        <img src=\"http://www.albumart.org/images/zoom-icon.jpg\"
+       alt=\"View larger image\" width=\"17\" height=\"15\"  border=\"0\"/></a>
```

Comment:
It makes my (old-Emacs-23) font-lock mode happy, and stops it from mistaking the whole rest of the file as quoted. I know it's an ugly kludge, though, so I don't mind leaving it out if you'd prefer.